var/spack/repos/builtin/packages/python/package.py

```diff
@@ -475,21 +485,46 @@ def write_easy_install_pth(self, exts):
                         "sys.__egginsert = p+len(new)\n")
 
     def activate(self, ext_pkg, **args):
+        path_view = args.get("path_view", "")
+
         ignore = self.python_ignore(ext_pkg, args)
         args.update(ignore=ignore)
 
         super(Python, self).activate(ext_pkg, **args)
 
-        exts = spack.store.layout.extension_map(self.spec)
+        # if we are activating in a filesystem view, we need to remove
+        # the link to the main easy_install.pth file prior to writing
+        # a custom one
+        if path_view:
```

Comment:
Mh… while I do fully agree with what you say I could have sworn I had the problems I described while writing the PR.

I guess I'll just remove change in question and once the [tests](https://github.com/LLNL/spack/pull/3227#discussion_r113430793) have been implemented we'll see if this is still an issue (which it should not be).